### PR TITLE
Fixes bug and adds more tests

### PR DESCRIPTION
**Pull request by edpin**
* Adds a few more tests for Organizations and Organizations Member
Invite.
* Fixes a bug where deleting an invitation should be a DELETE operation,
not POST.
* Adds coverage to the github workflow.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,3 +16,6 @@ pypitest: README.rst sdist
 test:
 	python -m unittest zanshinsdk/test_client.py
 
+coverage:
+	coverage run -m unittest zanshinsdk/test_client.py
+	coverage report
```
Comment (rmafra-tenchisec) [inline Makefile:21]: @edpin falta uma linha no fim do arquivo

Comment (edpin) [reply Makefile]: Fixed (not that it mattered).

File: zanshinsdk/test_client.py [hunk 2]
```diff
@@ -99,3 +128,42 @@ def test_delete_organization_members(self):
...
+        self.cli.delete_organization_member_invite('e22f4225-43e9-4922-b6b8-8b0620bdb110', 'john@example.com')
+
+        self.cli._request.assert_called_once_with(
+            'DELETE', '/organizations/e22f4225-43e9-4922-b6b8-8b0620bdb110/invites/john@example.com')
+
+    def test_resend_organization_member_invite(self):
+        self.cli.resend_organization_member_invite('822f4225-43e9-4922-b6b8-8b0620bdb1e3', 'juca@example.com')
+
+        self.cli._request.assert_called_once_with(
+            'POST', '/organizations/822f4225-43e9-4922-b6b8-8b0620bdb1e3/invites/juca@example.com/resend')
```
Comment (rfranco) [inline zanshinsdk/test_client.py:169]: Need a new line

**Review by rfranco — CHANGES_REQUESTED** [review]

File: zanshinsdk/test_client.py (hunk 2 above)
Comment (edpin) [reply zanshinsdk/test_client.py]: Fixed

**Review by rfranco — APPROVED** [review]

**Pull request merged**